googlesource/source.go

```diff
@@ -17,39 +17,50 @@ package googlesource
 import (
 	"context"
 	"errors"
+	"fmt"
 	"sync"
 	"time"
 
 	"cloud.google.com/go/bigquery"
 	sdk "github.com/conduitio/conduit-connector-sdk"
 	googlebigquery "github.com/neha-Gupta1/conduit-connector-bigquery"
-	bqStoragepb "google.golang.org/genproto/googleapis/cloud/bigquery/storage/v1"
+	"google.golang.org/api/option"
 	"gopkg.in/tomb.v2"
 )
 
 type Source struct {
 	sdk.UnimplementedSource
-	Session         *bqStoragepb.ReadSession
-	BQReadClient    *bigquery.Client
-	SourceConfig    googlebigquery.SourceConfig
-	Tables          []string
-	Ctx             context.Context
-	SDKResponse     chan sdk.Record
-	LatestPositions latestPositions
-	Position        Position
-	ticker          *time.Ticker
-	tomb            *tomb.Tomb
-	iteratorClosed  chan bool
+	bqReadClient *bigquery.Client
+	sourceConfig googlebigquery.SourceConfig
+	// haris: isn't this part of config?
+	// Neha : TableID is optional in config. If not provided we need to find them and then insert as
```

Comment:
Prior to merging, let's please clean up these comments (the ones from myself at least are good to go, not sure about Maha's).